format/writers.go

```diff
@@ -31,8 +31,9 @@ func NewFileWriter(filePath string) io.Writer {
 	return &fileWriter{filePath}
 }
 
+//nolint:gosec,gomnd // need to test this in a future PR
```

Comment:
```suggestion
//nolint:gosec,gomnd // TODO: AB#561861 need to test this in a future PR
```

@pacificcode this is what I meant. If todo make sure to note the todo. Azure board is automatically integrated with some of GitHub using AB# syntax and can even close the work items when you have in your PR "fixes AB#num"

Add the other comment please and note why bypassing gosec and we're good here. 

Thank you!